pdf/pdfcanvas.cpp

```diff
@@ -535,5 +582,123 @@ void PDFCanvas::resizeTimeout()
 void PDFCanvas::pageSizesFinished(const QList< QSizeF >& sizes)
 {
     d->pageSizes = sizes;
+    if (d->rememberWidth > 0) {
+      emit requestPageWidth(d->rememberWidth);
+    }
     layout();
+    if (d->rememberPage > 0 || d->rememberTop > 0. || d->rememberLeft > 0.) {
+      /* We wait for the layout to be done here, so pageRectangle() will
+         be available at QML level. */
+      emit requestPagePosition(d->rememberPage, d->rememberTop, d->rememberLeft);
+    }
+}
+
+/* Part related to access the local storage, as available from QML side. */
+static const QString dbConnection{"PdfSettings"};
+static QString _databasesPath()
```

Comment:
Ok, I didn't know about this. What do you suggest as a naming scheme for local static funtions ?